### PR TITLE
Feature/nfe related docs

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

File: l10n_br_fiscal/constants/fiscal.py [hunk 1]
```diff
@@ -328,7 +328,9 @@
 MODELO_FISCAL_CFE = "59"
 MODELO_FISCAL_CUPOM_FISCAL_ECF = "2D"
 MODELO_FISCAL_CTE = "57"
-MODELO_FISCAL_RL = "RL"
+MODELO_FISCAL_RL = "04"  # Produto Rural
```
Comment (marcelsavegnago) [inline l10n_br_fiscal/constants/fiscal.py]: https://github.com/OCA/l10n-brazil/pull/1188/files#diff-02723dceadf60070a3ab92bc1526a98b06670b21f95d9587ebbe58ec6eaf94a8R331

**Comment by mileo** [comment]
@marcelsavegnago @rvalyi @renatonlima can you check, it's ready for review.

**Comment by mileo** [comment]
@rvalyi @marcelsavegnago @renatonlima podem revisar daqui a pouco, testes rodando.

**Comment by mileo** [comment]
@rvalyi @renatonlima @marcelsavegnago Verdinho.

**Review by renatonlima — APPROVED** [review]

**Pull request merged**